akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -990,13 +1021,22 @@ private[stream] final class OrElse[T] extends GraphStage[UniformFanInShape[T, T]
 object GraphDSL extends GraphApply {
 
   class Builder[+M] private[stream] () {
+    private val unwiredIns = new mutable.HashSet[Inlet[_]]()
+    private val unwiredOuts = new mutable.HashSet[Outlet[_]]()
```

Comment:
Isn't worth the effort, they need to be converted later any way etc. It shouldn't be slower than the scala Maps we use in the `TraversalBuilder` anyways.